chamberInfo.py

```diff
@@ -1,5 +1,6 @@
 
 chamber_config = {
+    #0:"GE11-VI-L-CERN-0001"
```

Comment:
Maybe it's better to make a symlink and have three separate chamberInfo.py files for P5, coffin, and QC8?  This mirrors the functionality in cmsgemos with the connections file.  

Although this might increase code maintenance if we expand the dictionaries stored in chamberInfo.py, e.g. what Jared is suggesting in:

https://github.com/cms-gem-daq-project/vfatqc-python-scripts/issues/86